Mathlib/RingTheory/Subring/Units.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2021 Chris Birbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birbeck
```

Comment:
Can you mention where this came from in the PR description?

Reply:
I've done the archaeology myself, now in the description.

Reply:
This is surely "Birkbeck"?

Reply:
Oops, too late.

Reply:
#10966